core/modules/wiki.js

```diff
@@ -179,7 +179,7 @@ exports.generateNewTitle = function(baseTitle,options) {
 		title = baseTitle;
 	while(this.tiddlerExists(title) || this.isShadowTiddler(title) || this.findDraft(title)) {
 		title = baseTitle + 
-			(options.prefix || " ") + 
+			(options.prefix === undefined ? " " : options.prefix) + 
```

Comment:
I wonder if it might not be clearer still to test for `options.prefix` being a string; I think that that is really the intent here:

```
(typeof(options.prefix === "string") ? options.prefix : " ") + 
```